creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -86,52 +87,73 @@ const syncRouteController = async (req, res) => {
     )
   }
 
+  const wallet = walletPublicKeys[0]
+
   // If sync_type body param provided, log it (param is currently only used for logging)
   const syncType = req.body.sync_type
   if (syncType) {
     req.logger.info(
-      `SyncRouteController - sync of type: ${syncType} initiated for ${walletPublicKeys} from ${creatorNodeEndpoint}`
+      `SyncRouteController - sync of type: ${syncType} initiated for ${wallet} from ${primaryEndpoint}`
     )
   }
 
   /**
    * If immediate sync requested, enqueue immediately and return response
    * Else, debounce + add sync to queue
    */
+  const data = generateDataForSignatureRecovery(req.body)
+
   if (immediate) {
     try {
-      await processImmediateSync({
+      await processSyncOfTypeImmediate({
         serviceRegistry,
-        walletPublicKeys,
-        creatorNodeEndpoint,
-        forceResync
+        wallet,
+        creatorNodeEndpoint: primaryEndpoint,
+        forceResyncConfig: {
+          forceResync: req.body.forceResync,
+          apiSigning: {
+            timestamp: req.body.timestamp,
+            signature: req.body.signature,
+            data
+          },
```

Comment:
couple thoughts here
1. i know currently we are only enforcing signature on forceResync flag but logically these are 2 separate things bc we may later change the signature enforcement logic - would be better to have `forceResync` flag be separate from signature artifacts
2. the name we've used for this across codebase is `signatureData` ([CN example](https://github.com/AudiusProject/audius-protocol/blob/vg-force-resync-only-on-primary/creator-node/src/components/replicaSet/URSMRegistrationComponentService.js#L106)), best to stick to that instead of introducing new `apiSigning` name

Reply:
I dont understand what you are suggesting in point 1. this process function doesnt do the signing. the process function checks to see if forceResync is enabled. if not, continue. else, recover the wallet to verify identity. 

in point 2, I don't really see the point in changing the variable name especially since it's not used in actually signing/recovering data, but sure

Reply:
point 1 is that signatureData should not be nested inside `forceResyncConfig` since it is logically separate, thats all - nbd, we can change it in future if needed

and thx for changing name